Tools/CISupport/ews-build/events.py

```diff
@@ -135,7 +159,33 @@ def buildStarted(self, key, build):
             "builder_display_name": builder_display_name,
         }
 
-        self.sendData(data)
+        self.sendDataToEWS(data)
+
+    @defer.inlineCallbacks
+    def buildFinishedGitHub(self, build):
+        sha = self.extractProperty(build, 'revision')
+        repository = self.extractProperty(build, 'repository')
+
+        if not sha or not repository:
+            return
```

Comment:
Might be a good idea to print something to logs here, in case we have to debug a scenario when there is a problem with github communication.